### PR TITLE
components combined

### PR DESCRIPTION
**Pull request by nazi-pira**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/Components/CheckBox.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  }
+
+  return (
+    <label className="checkbox-label">
+      <input
+        type="checkbox"
+        checked={props.task.complete}
+        onChange={() => { handleCheckboxClick(props.task.id) }}>
+      </input>
+      <span className="checkmark" role="checkbox" aria-checked="false" tabIndex="0"></span>
```
Comment (Sartish) [inline code/src/Components/CheckBox.js:19]: Nice solution with handling the checkboxes to be complete⭐️

File: code/src/Components/ClearButton.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <>
+      <button className="remove-all-btn" type="button" onClick={() => { removeAllHandler() }} >
+        <span className="remove-all-btn-text" role="button" aria-label="remove">CLEAR ALL</span>
+      </button >
+      <Dialog
+        title="Do you want to clear all?"
+        supportingText="This will clear all task permanently and you can not undo it."
+        primaryBtnText="Yes"
+        secondaryBtnText="Cancel"
+        showDialog={allDialogs.dialog2}
```
Comment (Sartish) [inline code/src/Components/ClearButton.js]: I like that you added a Dialog that pops! The code looks clean and it is easy to understand it!

File: code/src/Components/Dialog.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  position: absolute;
+  left: calc(50% - 165px);
+  top: 250px;
+  width:320px;
+  padding:12px;
+  align-self:center;
+  box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2), 0px 1px 1px 0px rgba(0, 0, 0, 0.14), 0px 1px 3px 0px rgba(0,0,0,.12);
+  border-radius: 6px;
+  background: white;
+  z-index: 1;
```
Comment (Sartish) [inline code/src/Components/Dialog.js]: Love that you are using styled components! that box shadow is🙌

File: code/src/Components/Dialog.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+      dispatch(dialogs.actions.closeDialog())
+    }
+    if (btnText.toLowerCase() === "delete") {
+      dispatch(tasks.actions.removeTask())
+      dispatch(dialogs.actions.closeDialog())
+    }
+    if (btnText.toLowerCase() === "yes") {
+      dispatch(tasks.actions.removeAllTasks())
+      dispatch(dialogs.actions.closeDialog())
+    }
```
Comment (Sartish) [inline code/src/Components/Dialog.js]: very nice feature⭐️

File: code/src/Components/RemoveButton.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+
+export const RemoveButton = (props) => {
+  const dispatch = useDispatch()
+  const allDialogs = useSelector((store) => store.dialogs)
+
+  const handleRemoveTask = (id) => {
+    dispatch(dialogs.actions.openDialog())
+    dispatch(tasks.actions.setTaskToDelete(id))
+    // dispatch(tasks.actions.removeTask(id))
```
Comment (Sartish) [inline code/src/Components/RemoveButton.js]: don't forget to remove commeented out code ☺️

File: code/src/Components/header.css [hunk 6]
```diff
@@ -0,0 +1,31 @@
+.header-container {
+  padding: 18px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  /* border-bottom: solid #ffb677; */
```
Comment (Sartish) [inline code/src/Components/header.css:6]: same here, out-commented code can be removed ☺️

File: code/src/Components/removebuttons.css [hunk 7]
```diff
@@ -0,0 +1,34 @@
+.remove-all-btn {
+  margin-top: 5px;
+  padding: 15px;
+  /* background-color: #ff8364; */
```
Comment (Sartish) [inline code/src/Components/removebuttons.css:4]: here again to be removed 😄

File: code/src/reducers/tasks.js [hunk 8]
```diff
@@ -0,0 +1,48 @@
+import { createSlice } from '@reduxjs/toolkit'
+import moment from "moment"
+
+
+const taskData = [
+  { id: 1, text: "Stand-up with my team", complete: true, createdAt: "an hour ago" },
+  { id: 2, text: 'Demo the To-do app', complete: false, createdAt: "2 days ago", },
+  // { id: 3, text: 'Fork weekly assignment', complete: true },
+  // { id: 4, text: 'Update LinkedIn profile ', complete: true },
+  // { id: 5, text: 'Create a todo app', complete: false },
```
Comment (Sartish) [inline code/src/reducers/tasks.js:10]: maybe can be deleted also ? If not used for reference of course 😀

**Review by Sartish — APPROVED** [review]
Hi Nazanin! 
I am so impressed with your code! it was so easy to understand it, it is structured and clean🧹. I liked that you divided all the features in smaller components. Very impressive. Also, you used styled-components🙌 ⭐️ I will put the project as approved and the comments are only about commented code that can be removed 😄 But I could not see any link to your deployed project? Would be great if you put this in your readme file! I am also curious to see how it looks deployed 😍

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Please provide the netlify link

File: README.md [hunk 9]
```diff
@@ -1,12 +1,10 @@
...
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+A simple TODO app that can be used for adding new tasks and removing the old ones!
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I have faced probkems deploying the project on Netlify but with the help of my teammate got to solve it :)
 
 ## View it live
```
Comment (KarolCislowski) [inline README.md:10]: I still can't see a link to the hosted version here 😭

**Review by KarolCislowski — CHANGES_REQUESTED** [review]

**Pull request closed without merging**